packages/nextjs/components/Footer/FooterLinks.tsx

```diff
@@ -4,7 +4,7 @@ export const FooterLinks = () => {
   return (
     <div className="flex flex-col justify-between h-full w-full">
       <div className="flex md:pb-0 pb-[53px]">
-        <div className="flex md:pr-[100px] pr-[140px]">
+        <div className="flex md:pr-[100px] pr-[100px]">
```

Comment:
since these are the same padding values, you can just do `pr-[100px]` and remove `md:pr-[100px]`

Reply:
Thanks Sarah!